voidcrew/code/controllers/subsystem/overmap.dm

```diff
@@ -217,60 +222,51 @@ SUBSYSTEM_DEF(overmap)
 	for(var/i in 1 to CONFIG_GET(number/max_overmap_dynamic_events))
 		new /obj/structure/overmap/dynamic(get_unused_overmap_square_in_radius())
 
+/**
+ * ##get_ruin_list
+ *
+ * Returns the SSmapping list of ruins, according to the given desired ruin type
+ *
+ * Arguments:
+ * * ruin_type - a string, depicting the desired ruin type
+ */
+/datum/controller/subsystem/overmap/proc/get_ruin_list(ruin_type)
+	switch(ruin_type) // temporary because SSmapping needs a refactor to make this any better
```

Comment:
do you think this can be ruin_type.ruin_type which could be a var on ruin_type that would have SSmapping.[templates] instead?
Just an idea for further de-hardcoding

Reply:
Yeah, I'm going to redo this when i go through ssmapping (that's next)